pyfakefs/patched_packages.py

```diff
@@ -117,7 +117,6 @@ def __init__(self, _):
 
         class TextFileReader(parsers.TextFileReader):
             def __init__(self, *args, **kwargs):
-                kwargs["engine"] = "python"
```

Comment:
The whole patching of pandas (all what is currently inside the `if parsers is not None` conditions) can be removed for pandas >= 1.2. You can do something like this for the new condition:
```py
import pandas as pd
...

patch_pandas = (
    parsers is not None and [int(v) for v in pd.__version__.split(".")] < [1, 2, 0]
)
```

Also, please add an entry to the release notes - thanks!